### PR TITLE
#1103 JobScraper Added

### LINKED ISSUE #1103: Job Scraper From Indeed
Aim:
This python script will scrape jobs from indeed based on location and job title provided by the user and store it in a csv file.

@kaustubhgupta I am a GSSOC participant kindly assign me the same.

### PR DESCRIPTION
**Pull request by iamakkkhil**
# Description

- The python script uses BeautifulSoup to scrape job opening postings on indeed.com.
- The script takes user input for the required job title, location, the number of posts to collect and collect all the associated information.
- The information collected is stored in a `.csv` file.

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
 
## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] I have added the project meta data in the PR template.
- [x] I have created the ``requirements.txt`` file if needed.

## Project Metadata

`` If there is no-file/nothing to fill the below fields with, then type: none ``

`` Example: `` If no requirements.txt needed/present, then type none in ``Requirments``.

Category:
- [ ] Calculators
- [ ] AI/ML
- [x] Scrappers
- [ ] Social_Media
- [ ] PDF
- [ ] Image_Processing
- [ ] Video_Processing
- [ ] Games
- [ ] Networking
- [ ] OS_Utilities
- [ ] Automation
- [ ] Cryptography
- [ ] Computer_Vision
- [ ] Fun
- [ ] Others

Title: Job Scraper from Indeed

Folder: Job Scraper

Requirements: requirements.txt

Script: JobScraper.py

Arguments: None

Contributor: @iamakkkhil 

Description: The python script uses BeautifulSoup to scrape job opening postings on indeed.com.

### CODE REVIEW

**Review by santushtisharma10 — APPROVED** [review]
LGTM

**Review by Kushal997-das — APPROVED** [review]
LGTM :)

File: Job Scraper/JobScraper.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+
+def write_csv(loc, info):
+    """
+    The function writes the job openings collected in a .csv file
+    """
+    headers = ['Title', 'Company Name', 'Location', 'Date', 'Summary', 'Url']
+
+    # Adding info into the rows of the file
+    with open(loc+'_openings.csv', 'a', encoding='utf-8') as csv_f:
```
Comment (kaustubhgupta) [inline Job Scraper/JobScraper.py:16]:
```suggestion
    with open('./Job Scraper/' + loc+'_openings.csv', 'a', encoding='utf-8') as csv_f:
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by kaustubhgupta — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM..💯

**Pull request merged**